### PR TITLE
nautilus: doc: Backported dashboard documentation changes

### PR DESCRIPTION
**Pull request by LenzGr**
Backport of some dashboard-related documentation updates:

* doc: Updated dashboard iSCSI configuration, added labels #27074
* doc: dashboard: refined object gateway configuration #27581
* doc: Improved the dashboard proxy config section #27581
* doc: Fix small typo in dashboard documentation #27850

### CODE REVIEW

**Comment by ricardoasmarques** [comment]
jenkins render docs

File: doc/mgr/dashboard.rst [hunk 1]
```diff
@@ -316,12 +318,18 @@ The default value is 45 seconds.
...
-project. Please make sure that it's installed and enabled on the iSCSI gateways.
+The Ceph Dashboard can manage iSCSI targets using the REST API provided by the
+`rbd-target-api` service of the :ref:`ceph-iscsi`. Please make sure that it's
+installed and enabled on the iSCSI gateways.
+
+.. note::
+  The iSCSI management functionality of Ceph Dashboard depends on the latest
+  version 3 of the `ceph-iscsi <https://github.com/ceph/ceph-iscsi>`_ project.
+  Make sure that your operating system provides the correct version, otherwise
+  the dashboard won't enable the managemement features.
```
Comment (s0nea) [inline doc/mgr/dashboard.rst]:
```suggestion
  the dashboard won't enable the management features.
```

**Comment by s0nea** [comment]
@LenzGr I created a PR against `master` https://github.com/ceph/ceph/pull/27850 so that you can cherry-pick (if you want to) the typo fix as soon as the PR has been merged.

File: doc/mgr/dashboard.rst (hunk 1 above)
Comment (LenzGr) [reply doc/mgr/dashboard.rst]: Fixed by cherry-picking #27850 into this PR.

**Comment by LenzGr** [comment]
jenkins render docs

**Comment by LenzGr** [comment]
jenkins run make check

**Comment by LenzGr** [comment]
retest this please

**Comment by LenzGr** [comment]
jenkins test docs

**Pull request merged**